.github/workflows/ubuntu.yml

```diff
@@ -42,14 +43,25 @@ jobs:
           libxcb-dri3-dev \
           libxext-dev \
           libxfixes-dev \
-          libwayland-dev
+          libwayland-dev \
+          meson
```

Comment:
* https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md
* https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md

Interesting, they indeed does not seem to include meson by default. At least according to readme description.

Reply:
Many distros do not include meson in the normal build tool-chain - not sure why.